python/ray/data/datasource/parquet_base_datasource.py

```diff
@@ -14,10 +14,12 @@
     FileBasedDatasource,
     _resolve_kwargs,
 )
+from ray.util.annotations import DeveloperAPI
 
 logger = logging.getLogger(__name__)
 
 
+@DeveloperAPI
```

Comment:
This is the Parquet datasource used for bulk Parquet reading, and can be used directly by the end-user just like the `ParquetDatasource`; should this be a public API?

Reply:
I see, changed.